core/templates/pages/topic-editor-page/questions-tab/topic-questions-tab.directive.ts

```diff
@@ -85,6 +87,13 @@ angular.module('oppia').directive('questionsTab', [
             );
           };
           ctrl.$onInit = function() {
+            // To set autofocus when screen loads.
+            $window.onload = function() {
+              FocusManagerService.setFocus('selectSkillField');
```

Comment:
Does this work if it takes a while for the component to be visible on the screen?

Reply:
yes it does as $window.onload function fires up only after the page has completely loaded